Source/Initialization/WarpXInitData.cpp

```diff
@@ -346,8 +341,7 @@ WarpX::InitLevelData (int lev, Real /*time*/)
               Efield_aux[lev][i]->setVal(E_external_grid[i]);
               Efield_cp[lev][i]->setVal(E_external_grid[i]);
               if (fft_do_time_averaging) {
-                  Efield_avg_aux[lev][i]->setVal(E_external_grid[i]);
-                  Efield_avg_cp[lev][i]->setVal(E_external_grid[i]);
+                Efield_avg_cp[lev][i]->setVal(E_external_grid[i]);
```

Comment:
I think our standard four whitespaces became only two here, by mistake:
```suggestion
                  Efield_avg_cp[lev][i]->setVal(E_external_grid[i]);
```